src/main/java/org/openrewrite/java/testing/jmockit/JMockitBlockRewriter.java

```diff
@@ -131,6 +132,11 @@ private void rewriteMethodInvocation(List<Statement> statementsToRewrite) {
             rewriteResult(invocation, mockInvocationResults.getResults());
         }
 
+        if (blockType == NonStrictExpectations) {
```

Comment:
What happens in JMockit if `NonStrictExpectations` block contains a `times = 2` statement? Is that not verified?

Reply:
Yes this is only for stubbing, there is link https://javadoc.io/doc/org.jmockit/jmockit/1.22/mockit/NonStrictExpectations.html

Reply:
It's a bit unclear what the behavior is, due to this statement:

"The difference from [regular expectations](https://javadoc.io/static/org.jmockit/jmockit/1.22/mockit/Expectations.html) is that a non-strict recorded expectation is allowed to have no matching invocations **(unless otherwise specified).**"

I wonder if this means that if `times` is specific, does it get honored? I'm fine with making this assumption for now but I wouldn't be surprised if JMockit has the ability to mix both.